railties/lib/rails/generators/test_unit/authentication/templates/test/controllers/passwords_controller_test.rb.tt

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+
+class PasswordsControllerTest < ActionDispatch::IntegrationTest
+  test "new" do
+    get new_password_path
+    
+    assert_response :success
+  end
+
+  test "create" do
+    post passwords_path, params: { email_address: users(:one).email_address }
+
+    assert_enqueued_email_with PasswordsMailer, :reset, args: [ users(:one) ]
+    assert_redirected_to new_session_url
+    follow_redirect!
+    assert_select "div", /reset instructions sent/
```

Comment:
As an alternative to following the redirect and asserting the flash as the contents of a `<div>` element, have you considered asserting against [flash.notice](https://edgeapi.rubyonrails.org/classes/ActionDispatch/Flash/FlashHash.html#method-i-notice)?

```suggestion
    assert_equal "Password reset instructions sent (if user with that email address exists).", flash.notice
```